### DISCUSSION TITLE
提供几个残局

### DISCUSSION THREAD

**Discussion by wsy998** [official-pikafish/Pikafish#55] [General]
n2ak4/4a4/2P6/1N7/9/9/6P1P/C1N1BK3/6p2/R1B1r4 w - - 0 1
还有一个是万里长征这个残局，皮卡鱼对这种长距离运兵貌似不能计算到

**Comment by xiangqishare** [comment]
存在海量的排局或残局，都很难计算，或者计算不到。
这是正常现象，引擎提升不以特殊局面为目标。
